tools/keyvault-mock-attestation/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node:14-alpine
+FROM mcr.microsoft.com/cbl-mariner/base/nodejs:16
```

Comment:
Testing in local and passed.

Reply:
How did you test? Just building the DockerFile isn't enough because we need to be sure that whoever uses it still continues to work as expected.

Reply:
I can take a look at it this morning 😄 will approve after testing